pkg/migrations/internal/seq_num.go

```diff
@@ -6,5 +6,5 @@ var (
 	// It is a shared constant between central and the migrator binary.
 	CurrentDBVersionSeqNum = 112
 	// PostgresDBVersionPlus is the current DB version number with Postgres DB data migration.
-	PostgresDBVersionPlus = 56
+	PostgresDBVersionPlus = 58
```

Comment:
Why not 57? Is it because it was not bumped last time?

Reply:
This is the 57th migrations in the n_ series, so it should be 57. #2630 was in before I hooked up Postgres migrations.

Reply:
Then the files should also be named 56_to_57.

Reply:
No, it should be named 57_to_58 and PostgresDBVersionPlus is 57. We have test case to cover if the number is wrong.

Reply:
I understand why you think it should be 56_to_57. You may have noticed n_ series is 1-based not 0-based. I should have changed it.